qln/msghandler.go

```diff
@@ -14,49 +14,47 @@ import (
 
 func (nd *LitNode) registerHandlers() {
 
-	mp := nd.PeerMan.GetMessageProcessor()
```

Comment:
Is there a reason we can't just have a reference on the stack to the mproc?  `DefineMessage` calls it on a reference already anyways.

Reply:
There is no pass by reference in go :)

Reply:
We're directly reaching deeply into structs again here which makes things complicated to refactor.  In this case it doesn't matter since I don't see how we keep track of the message processor changing in the future but imho it looks messier to do the whole `nd.PeerMan.MProc` dance.

But also pass by reference (done differently here):
```go
func (thing *Thing) SomeFunc(obj *OtherThing) {
}
...
var t Thing
var o OtherThing
t.SomeFunc(&o)
```

Reply:
Yeah, we could do the pointer hack, but its not something that the go pundits recommend, idk

Reply:
that's not a hack...